src/server/api/routes/days-to-end-batch.router.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable no-console */
+const express = require('express');
+
+const router = express.Router({ mergeParams: true });
+
+// controllers
+const dayLeftController = require('../controllers/days-to-end-batch.controllers');
+
+// ENDPOINT: /api/day-left/:batchId :GET to get Days left to end the batch
+router.get('/:batchId', (req, res, next) => {
```

Comment:
its good to protect the endpoint with a middleware.Could you refer to firebase-middleware.md file in our project on how to use middleware on endpoint